dv/uvm/riscv_dv_extension/testlist.yaml

```diff
@@ -282,3 +282,22 @@
     +directed_instr_2=riscv_multi_page_load_store_instr_stream,20
     +enable_unaligned_load_store=1
   rtl_test: core_ibex_base_test
+
+- test: riscv_mem_error_test
+  description: >
+    Normal random instruction test, but randomly insert instruction fetch or memory load/store errors
+  iterations: 5
+  gen_test: riscv_rand_instr_test
+  gen_opts: >
+    +require_signature_addr=1
+    +instr_cnt=10000
+    +directed_instr_0=riscv_load_store_rand_instr_stream,4
+    +directed_instr_1=riscv_loop_instr,4
+    +directed_instr_2=riscv_hazard_instr_stream,4
+    +directed_instr_3=riscv_load_store_hazard_instr_stream,4
+    +directed_instr_4=riscv_multi_page_load_store_instr_stream,4
```

Comment:
All the directed instructions seems to be redundant as it's injected by riscv_rand_instr_test by default

Reply:
Removed.